Week2/homeworkWeek2.js

```diff
@@ -0,0 +1,125 @@
+const mysql = require('mysql');
+const util = require('util');
+const prompt = require('prompt');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+});
+
+const execQuery = util.promisify(connection.query.bind(connection));
+const input = util.promisify(prompt.get.bind(this));
+
+connection.connect(error => {
+  if (error) throw error;
+  console.log('\nMysql connected...\n');
+});
+
+const help = () => {
+  console.log(`
+    To answer one of the following questions, insert the question number:
+    1 ..... What is the capital of country X ? (Accept X from user)
+    2 ..... List all the languages spoken in the region Y (Accept Y from user)
+    3 ..... Find the number of cities in which language Z is spoken (Accept Z from user)
+    4 ..... Accept the region and language from the user. Are there any countries in this 
+            region with the given language as the official language ? If yes, display those countries. 
+            If no, display FALSE. E.g. (A) input region : 'Western Europe' and input language : 
+            'Dutch' output should be Belgium and Netherlands 
+            (B) input region : 'Western Europe' and input language : 'Hindi' output should be 'FALSE'
+    5 ..... List all the continents with the number of languages spoken in each continent
+  `);
+};
+
+async function queryDatabase() {
+  help();
+  prompt.start();
+  try {
+    const use_database = `use world`;
+    await execQuery(use_database);
+
+    const options = await input(['number']);
+    const inputNumber = options.number;
+
+    switch (inputNumber) {
+      case '1':
```

Comment:
I like the use of switch case, but user has to run the program multiple times to execute different queries. After every input, the program terminates. Writing a menu driven program would be a good idea. (having a while loop around the switch case and providing an option for executing) Check this link to know more about menu driven program : https://www.geeksforgeeks.org/menu-driven-program-using-switch-case-c/